src/services/config/service.ts

```diff
@@ -35,9 +35,15 @@ export const makeConfig = ({
       env.STAKING_MODULE_ID,
       'Please, setup STAKING_MODULE_ID id. Example: 123'
     ),
-    OPERATOR_ID: num(
-      env.OPERATOR_ID,
-      'Please, setup OPERATOR_ID id. Example: 123'
+    OPERATOR_ID: optional(() =>
+      num(env.OPERATOR_ID, 'Please, setup OPERATOR_ID id. Example: 123')
+    ),
+    OPERATOR_IDENTIFIERS: optional(() =>
```

Comment:
Maybe just `OPERATOR_IDS`?

And instead of validating below do
```ts
env.OPERATOR_IDENTIFIERS = env.OPERATOR_IDENTIFIERS
  ? json_arr(
      env.OPERATOR_IDENTIFIERS,
      (oracles) => oracles.map(num),
      'Please, setup OPERATOR_IDENTIFIERS. Example: [1,2,3]'
    )
  : [OPERATOR_ID];
```

Reply:
fixed